README.md

```diff
@@ -1,6 +1,6 @@
-| CicleCI Build Status | Sonar Code Quality |
-|----------------------|---------------------|
-[![CircleCI](https://circleci.com/gh/usdot-fhwa-stol/carma-platform.svg?style=svg)](https://circleci.com/gh/usdot-fhwa-stol/carma-platform) | [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=usdot-fhwa-stol_CARMAPlatform&metric=alert_status)](https://sonarcloud.io/dashboard?id=usdot-fhwa-stol_CARMAPlatform) |
+| CicleCI Build Status | Sonar Code Quality | DockerHub Build Dev | DockerHub Candidate | DockerHub STOL |
```

Comment:
I'd flip the order here, maybe go "DockerHub Release | DockerHub Release Candidate | DockerHub Develop"

Reply:
Changed to correct order and updates names